e2e/backend.e2e.spec.ts

```diff
@@ -89,6 +89,17 @@ test.describe.serial('KubernetesBackend', () => {
     test('should detect changes', async() => {
       const currentSettings = (await get('/v1/settings')) as Settings;
 
+      if (!currentSettings.kubernetes.version) {
+        // The Kubernetes version could be empty if it's previously disabled.
+        // Set something.
+        const updatedSettings: RecursivePartial<Settings> = {
+          kubernetes: { version: '1.23.4' },
+          version:    CURRENT_SETTINGS_VERSION,
```

Comment:
It seems wrong to use the current settings version; if we move the Kubernetes version to a different place in the schema, then this code will be broken. Since we have version migration now even for API calls, setting this to a fixed version that corresponds to the schema being used here will remain valid even when the schema changes in the future.
```suggestion
          version:    10,
```

Reply:
Force-pushed to use a fixed version.